torch/_inductor/codecache.py

```diff
@@ -917,20 +917,26 @@ def install_gcc_via_conda() -> str:
 
 
 def is_gcc() -> bool:
+    # Mac OS not support real gcc, it is acturally a clang.
+    if sys.platform == "darwin":
```

Comment:
This feels like a bit more correct behavior to me (I've seen a few issues when users installed gcc on Apple platform from anaconda. Even Inductor still has a logic to do so
```suggestion
    if sys.platform == "darwin" and is_apple_clang():
```